Imperatus.rte/Actors/Shared.ini

```diff
@@ -25,5 +25,5 @@ AddEffect = AEmitter
 	NegativeThrottleMultiplier = 0.5
 	PositiveThrottleMultiplier = 0.9
 	ParticlesPerMinute = 9600
-	BurstSize = 10
+	BurstSize = 9
```

Comment:
What's the reason for the jetpack nerfs? Trying to make heavier actors less mobile??

Reply:
I tested out the Impie jetpacks and they felt ridiculously powerful to me, I must be biased due to experience though... I'll take a closer look